nuxeo-drive-client/nxdrive/wui/conflicts.py

```diff
@@ -32,6 +117,7 @@ def get_errors(self):
 
     @QtCore.pyqtSlot(result=str)
     def get_conflicts(self):
+        log.warn("Retrieving conflicts ...")
```

Comment:
When a new conflict happens after users are retrieved for the day, the new conflict will encounter a new userid which  is not yet resolved. This will take us back to original problem where conflict window will show blank page until the userid for recent conflict is resolved